frontend/javascripts/viewer/model/bucket_data_handling/bucket_traversals.ts

```diff
@@ -29,7 +29,7 @@ export default function traverse(
   const voxelSize = getBucketExtent(mags[zoomStep]);
   // In addition, the variables stepX and stepY are initialized to either 1 or -1 indicating whether X and Y are
   // incremented or decremented as the ray crosses voxel boundaries (this is determined by the sign of the x and y components of → v).
-  const [stepX, stepY, stepZ] = v.map((el) => Math.sign(el));
+  const [stepX, stepY, stepZ] = v.map((el: number) => Math.sign(el));
```

Comment:
why is this change necessary?

Reply:
<img width="519" height="79" alt="image" src="https://github.com/user-attachments/assets/2bc2586b-d11b-46ac-b3d7-4f4484a71759" />   

I dont know why the typechecker is complaining there. `v` is the result of `V3.sub: function (x: Vector3, y: Vector3, res?: Vector3): Vector3 {}`

Reply:
I had another look and apparently TS does a poor job here, because the mjs d.ts file was missing a definition (or import) for `Vector3`. therefore, it assigned `any` to `Vector3`, but didn't escalate this. I added the Vector3 type to the definition module to fix this (so, no explicit `Vector3` is necessary for `v`). importing the type in d.ts would be nicer, obviously, but I didn't feel like figuring that out now. it's only one line, so, it should not hurt.